jest.config.js

```diff
@@ -25,4 +25,18 @@ module.exports = {
   globals: {
     __DEV__: true,
   },
+
+  // reporter for circleci
+  reporters: [
+    'default',
+    [
+      'jest-junit',
+      {
+        outputDirectory: 'junit/jest',
```

Comment:
I find it strange that the e2e equivalent of this file `wdio.saucelabs.conf.js` is adds `__dirname` to `outputDirectory`. I'm not familiar enough to know if it's a typo or done on purpose.

Reply:
they just use different configuration formats, jest assumes surname, while wdio wants an absolute path